File: Python/HelloThread/HelloWorldThread.py
import threading

class myThread(threading.Thread):
    def run(self):
        for i in range(10):
            print(threading.currentThread().getName() + "-- Hello World")

def main():
    thread1 = myThread()
    thread2 = myThread()

    thread1.start()
    thread2.start()

if __name__ == '__main__':
    main()

File: Java/HelloThread/src/HelloWorldThread.java
public class HelloWorldThread extends Thread {
    public void run() {
        for(int i =0; i<10; i++){
            System.out.println (getName() + "-- Hello World");
        }
        
    }
    public static void main(String [] args) {
        HelloWorldThread t = new HelloWorldThread () ;
        HelloWorldThread t2 = new HelloWorldThread () ;
        t.start();
        t.setName("Thread 1");
        t2.start();
        t.setName("Thread 2");
    }
}